packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -107,11 +139,22 @@ class _ScaffoldGeometryNotifier extends ChangeNotifier implements ValueListenabl
     double bottomNavigationBarTop,
     Rect floatingActionButtonArea,
     double floatingActionButtonScale,
+    NotchMaker floatingActionButtonNotchMaker,
   }) {
     fabScale = floatingActionButtonScale ?? fabScale;
+    geometry = geometry.copyWith(
+      bottomNavigationBarTop: bottomNavigationBarTop,
+      floatingActionButtonArea: floatingActionButtonArea,
+      floatingActionButtonNotchMaker: floatingActionButtonNotchMaker,
+    );
+    notifyListeners();
+  }
+
+  void _updateFabNotchMaker(NotchMaker fabNotchMaker) {
```

Comment:
nit: avoid abbreviations

Reply:
Done, though this one is really becoming too long `_updateFloatingActionButtonNotchMaker`, and is a private method so I would think it is reasonable to use FAB.

There's a tradeoff between including all the details everywhere and not being over verbose, and I'm not sure it is correct to always prefer to error on the more information side.

Reply:
Also we now have a `_setFloatingActionButtonNotchMakerAndNotify` method... it is only used in the same class, and called from 4 lines above it's declaration. I think using an abbreviation in this case is the right choice.